features/product/managing_product_variants/disable_or_enable_inventory_on_product_variant.feature

```diff
@@ -0,0 +1,28 @@
+@managing_product_variants
+Feature: Enable the inventory tracking
+    In order to enable inventory tracking
+    As an Administrator
+    I want to be able to disable or enable inventory tracking
```

Comment:
Oh, ok. The feature title here is `Enable the inventory tracking` but feature file name suggest both actions `disable_or_enable_inventory_on_product_variant`.